python/sidecar/http.py

```diff
@@ -81,17 +80,19 @@ async def http_search_s2(req: SearchRequest) -> SearchResponse:
 @ai_api.post("/rewrite")
 async def http_ai_rewrite(
     req: RewriteRequest,
-    user_settings: Annotated[SettingsSchema, Depends(settings.get_settings_for_user)],
+    # user_settings: Annotated[SettingsSchema, Depends(settings.get_settings_for_user)],
```

Comment:
delete these lines if they're no longer needed

Reply:
Right. I had them to discuss with Greg.